juju/juju
# WIP: State: Add type field to application

## Description of change

Add type field to Application doc, distinguishing between IAAS and CAAS applications.

Blocked because we first need to merge https://github.com/juju/description/pull/27 and update dependencies here with the resulting commit.

## QA steps

Basic bootstrap/deploy functionality check creates an application and gets at these changes.

## Documentation changes

No effects on workflow.
